README.md

```diff
@@ -17,6 +17,26 @@ features Configuration Service provides APIs for subscriber management.
 4. 5G clients can connect & get complete configuration copy through grpc interface.
 5. 4G clients communicate with Webconsole through REST interface
 
+# UI
+
+Webconsole can optionally serve static files, which constitute the frontend part of the application.
+
+This functionality is not mandatory. The frontend files must be provided and placed in `webconsole/ui/frontend_files` before compilation, as they are not included in the webconsole codebase.
+
+To compile the webconsole including the UI option:
+```
+make webconsole-ui
+```
+or
+```
+go build --tags ui -o <output-file> ./server.go
+```
+
```

Comment:
I would avoid providing options here to keep it simple for contributors, I'd say select one or the other.